packages/react-renderer-demo/src/app/pages/renderer/component-mapping.md

```diff
@@ -7,26 +7,16 @@ import ListOfContents from '../../src/helpers/list-of-contents';
 <Grid container item>
 <Grid item xs={12} md={10}>
 
-# Introduction
+# ComponentMapper
```

Comment:
There are `title` and `description` attributes in schema that are no longer rendered automatically. Can you move these to the form template example?